wpilibNewCommands/src/main/native/include/frc2/command/PerpetualCommand.h

```diff
@@ -43,14 +45,15 @@ class PerpetualCommand : public CommandHelper<CommandBase, PerpetualCommand> {
    *
    * @param command the command to run perpetually
    */
-  WPI_DEPRECATED(
+  [[deprecated(
       "PerpetualCommand violates the assumption that execute() doesn't get "
       "called after isFinished() returns true -- an assumption that should be "
-      "valid."
-      "This was unsafe/undefined behavior from the start, and RepeatCommand "
-      "provides an easy way to achieve similar end results with slightly "
-      "different (and safe) semantics.")
-  explicit PerpetualCommand(std::unique_ptr<Command>&& command);
+      "valid. This was unsafe/undefined behavior from the start, and "
+      "RepeatCommand provides an easy way to achieve similar end results with "
+      "slightly different (and safe) semantics.")]] explicit PerpetualCommand(std::
+                                                                                  unique_ptr<
+                                                                                      Command>&&
+                                                                                      command);
```

Comment:
Maybe we keep the WPI_DEPRECATED macro usage for constructors until this is fixed in clang-format?